game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/EditorPanel.java

```diff
@@ -1,34 +0,0 @@
-package games.strategy.engine.framework.startup.ui.editors;
-
-import java.awt.Color;
-import java.awt.GridBagLayout;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-
-/**
- * Helper Base class for Editors, that provides a basic collection of useful operations.
- *
- * <p>This class used to have a legitimate reason to be a superclass of editors, but making all
- * methods in this class static, and converting it to a utility class is probably a good thing to
- * do.
- */
-abstract class EditorPanel extends JPanel {
-  private static final long serialVersionUID = 8156959717037201321L;
-  private final Color labelColor = new JLabel().getForeground();
-
-  EditorPanel() {
-    super(new GridBagLayout());
```

Comment:
Layout is moved to children classes, 'setLabelValid' is moved to SwingComponents. The two changes eliminates the need for this base-class.